qos-monitor/src/main/resources/log4j2.xml

```diff
@@ -5,7 +5,7 @@
             %d{yyyy-MM-dd HH:mm:ss.SSS} %5p ${hostName} --- [%15.15t] %-40.40c{1.} : %m%n%ex
         </Property>
 		<Property name="JDBC_LEVEL">INFO</Property>
-		<Property name="CONSOLE_FILE_LEVEL">INFO</Property>
+		<Property name="CONSOLE_FILE_LEVEL">DEBUG</Property>
```

Comment:
should be INFO when we merge into development

Reply:
done